src/Orleans/IDs/ActivationAddress.cs

```diff
@@ -32,6 +33,7 @@ internal class ActivationAddress
         public GrainId Grain { get; private set; }
         public ActivationId Activation { get; private set; }
         public SiloAddress Silo { get; private set; }
+        public MultiClusterStatus Status { get; private set; }
```

Comment:
There seems to be at least one case where the status has to be serialized (namely, when handing off activations). I have updated the binarystreamreader and binarystreamwriter and the serialization unit test accordingly. This adds one byte.